api/src/main/resources/workbench.yaml

```diff
@@ -888,6 +888,228 @@ paths:
+          name: standard_concept
+          type: string
+          required: false
+          description: standard concept filter
+        - in: query
+          name: concept_code
+          type: string
+          required: false
+          description: concept code  filter
+        - in: query
+          name: vocabulary_id
+          type: string
+          required: false
+          description: vocabulary id filter
+        - in: query
+          name: domain_id
+          type: string
+          required: false
+          description: vocabulary id filter
+      responses:
+        "200":
+          description: A collection of concepts
+          schema:
+            $ref: "#/definitions/ConceptListResponse"
+
+  /api/v1/databrowser/analysis-results:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Gets results for an analysis id and stratum
+      operationId: "getAnalysisResults"
+      parameters:
+        - in: query
+          name: analysis_id
+          type: integer
+          format: int64
+          required: true
+          description: analysis_id
+        - in: query
+          name: stratum_1
+          type: string
+          required: false
+          description: stratum 1
+        - in: query
+          name: stratum_2
+          type: string
+          required: false
+          description: stratum 2
+        - in: query
+          name: stratum_3
+          type: string
+          required: false
+          description: stratum 3
+        - in: query
+          name: stratum_4
+          type: string
+          required: false
+          description: stratum 4
+        - in: query
+          name: stratum_5
+          type: string
+          required: false
+          description: stratum 5
+
+      responses:
+        "200":
+          description: A collection of analysis results from achilles_results_view
+          schema:
+            $ref: "#/definitions/AnalysisResultListResponse"
+
+  /api/v1/databrowser/analyses:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Gets list of analysis definitions
+      operationId: "getAnalyses"
+      parameters:
+      responses:
+        "200":
+          description: A collection of analysis definitions
+          schema:
+            $ref: "#/definitions/AnalysisListResponse"
+
+  /api/v1/databrowser/participant-count:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Gets results for an analysis id and stratum
+      operationId: "getParticipantCount"
+      parameters:
+      responses:
+        "200":
+          description: A collection of analysis results from achilles_results_view
+          schema:
+            $ref: "#/definitions/AnalysisResult"
+
+  /api/v1/databrowser/concept-count:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Gets count for concept
+      operationId: "getConceptCount"
```

Comment:
Can you explain when you'd use getConceptCount vs. getParticipantCount?

Reply:
concept count is counts for a concept . participant count is how many total participants we have data for.